spark/src/main/java/org/apache/iceberg/spark/source/RowDataReader.java

```diff
@@ -194,20 +195,32 @@ protected Schema tableSchema() {
   }
 
   private static UnsafeProjection projection(Schema finalSchema, Schema readSchema) {
-    StructType struct = SparkSchemaUtil.convert(readSchema);
+    StructType readStruct = SparkSchemaUtil.convert(readSchema);
```

Comment:
Renamed these variables because too many things were called "struct, or ref" and I was getting confused which was which

Reply:
Are these changes still needed?